src/structurizr/workspace.py

```diff
@@ -44,6 +44,7 @@ class WorkspaceIO(BaseModel):
         name (str): The name of the workspace.
         description (str): A short description of the workspace.
         version (str): A version number for the workspace.
+        revision (int): The internal revision number.
```

Comment:
According to the OpenAPI schema (https://editor.swagger.io/?url=https://raw.githubusercontent.com/structurizr/json/master/structurizr.yaml) the revision should **not** be part of the JSON? Maybe it's wrong?

Reply:
It's a great idea to add an integration testing to validate using this schema. Opened a ticket: #20

Reply:
And regarding the revision. It returns in the response and then we populate the WorkspaceIO.